base-server/index.js

```diff
@@ -666,6 +680,36 @@ export class BaseServer {
     this.debugActionError(meta, `Wrong channel name ${action.channel}`)
   }
 
+  onAction(action, meta, clientId) {
+    let queueType
+    if (!this.queueTypes[0]) {
+      queueType = 'main'
+    }
+    else {
+      let type = action.type.slice(0, action.type.indexOf('/'))
```

Comment:
We should not use `/` magic:

1. It is not part of Logux Server semantic. For server, every `action.type` is just a string.
2. We are making DX more complicated. It is better to explicitly define `queue` for each type, rather of explaining complex magic logic.

Reply:
What would be a good way to place `logux/` actions into a correct queue?
If developer creates a queue for a particular type, but not entire channel, we may not have enough information.  One way I see is to store action ids to reliably place `logux/subscribe` to correct queue, but to me it seems it would require even more complex logic.

Reply:
User will define queue for channels: `server.channel(…, { queue: 'users' })`

If action processing, we will check `action.type === 'logux/subscribe || …` explicitly and use `action.channel` to get `queue` from `server.channel` definitions.

Reply:
Should we allow user to place a particular `type` to a separate queue? If such ability is not needed, then this is simple

Reply:
> Should we allow user to place a particular type to a separate queue?

Yes, each `type` must explicitly has queue. No auto-generation for queue because queues should be explicit.